src/components/icon/index.css

```diff
@@ -0,0 +1,12 @@
+.icon {
+    display: flex;
+    margin: 5px;
```

Comment:
maybe we can create a css for the margins and padding like the font-sizes file and colors file